Apps/Inc/Tasks.h

```diff
@@ -152,6 +158,20 @@ typedef enum{
     FAULT_BPS = 0x20,       // for if BPS trips
 } fault_bitmap_t;
 
+/**
+ * Task trace
+ * 
+ * Stores the last 8 tasks that were run
+ * The most recent task is at tasks[index], the one before at tasks[index-1],
+ * wrapping back around at the beginnning
+ */
+typedef struct {
+    OS_TCB *tasks[8];
+    uint32_t index;
```

Comment:
Doesn't matter but change to uint8_t to save space.

Reply:
Due to padding it actually wouldn't make a difference.